io/eolearn/tests/test_io.py

```diff
@@ -262,11 +264,33 @@ def setUpClass(cls):
                     instance_id=instance_id
                 ),
                 bbox=bbox,
-                time_interval=cls.time_interval,
+                time_interval=long_time_interval,
                 feature_type=FeatureType.MASK
             ),
         ]
 
+        cls.task_cases_image_format = [

```

Comment:
This test case is not used anywhere.

Reply:
It is the only one with defined `image_format=MimeType.TIFF_d32f`. I guess I forgot to add it to list of `test_cases`.